file-processing/main.bal

```diff
@@ -0,0 +1,86 @@
+import ballerina/file;
+import ballerina/io;
+import ballerina/log;
+import ballerina/sql;
+import ballerinax/mysql;
+import ballerinax/mysql.driver as _;
+
+const string CSV_EXT = ".csv";
```

Comment:
Do we need to specify the type when declaring consts?

Reply:
Not necessary. But IMO, better to define the type for consts as a best practice